flytekit/types/file/file.py

```diff
@@ -272,6 +273,10 @@ def to_literal(
         if python_val is None:
             raise TypeTransformerFailedError("None value cannot be converted to a file.")
 
+        # Correctly handle `Annotated[FlyteFile, ...]` by extracting the origin type
+        if typing.get_origin(python_type) is Annotated:
```

Comment:
`get_origin` should also import from typing_extensions

Reply:
Pesky 3.8 compatibility, sorry for the additional turnaround - have been using 3.9+ exclusively for some time, so my mental radar doesn't catch these.

Fixed for `get_origin` and `get_args` in bffa9b8a.